### DISCUSSION TITLE
Beaten Section Ideas

### DISCUSSION THREAD

**Discussion by Player1041** [RetroAchievements/RAWeb#1829] [Ideas]
So hearing there is a beaten section in the works, I have some suggestions for the feature.

- Badge icons, such as the ones used for mastery, being available *possibly* in a different section to where they are now.
- In addition to these icons, they would have a bronze border, as they are the lowest of 3 stages of completion. 
- Alongside the border changes, the softcore border could be changed from blue to silver to reflect the fact that it's 2nd place, it's between bronze and gold, it would make aesthetic sense in my opinion.
- Also, as to not clog the sidebar, the option of `View More` being available when the user has maybes 50+ badges, though this number could be raised or lowered depending on what you (the site team) deem a good number to cause the button to exist at.

**Comment by StingX2** [comment]
There are going to be NO badges for beaten games

**Comment by Player1041** [comment]
How come? Just to prevent like miscommunication, I mean these by badges. https://i.vgy.me/PzZJan.jpg

**Reply by drisc** [reply]
Sorry I should have clarified this yesterday, the progression system doesn't use a new type of badge. Introduces a new page with more detailed information about your progress within games (beaten, completed, mastered, time taken etc).

**Comment by wescopeland** [comment]
No plans for badges at this time, but there are other ways we can incentivize making pushing your beaten games count higher and higher something worth doing.

**Reply by Player1041** [reply]
Just out of curiosity, what are some of those incentives if you have any planned and are fine with sharing them?

**Reply by wescopeland** [reply]
I'm afraid to say too much because things are very subject to change, but a lot of time and effort is being put into this to make sure everyone has the best experience we can build 🙏

**Reply by Player1041** [reply]
That's fair enough, hope it turns out well :)

**Reply by Kinglink** [reply]
Honestly a number + list going up sounds amazing.

Progress tracking could be fun, if you're tackling it, but I just want a list of games I've "Finished" and to me getting to the end of the story is "Finishing" a game normally.

**Comment by Alexdatadestroyer** [comment]
Using a perspective that we use a yellow square for Set Badges in case of Mastered (Hardcore) and none in case of Completed (Softcore), considering that Beaten status is only given when played on Hardcore Mode, then, how about using a color that resembles a silver square followed by a black and white version of Set Badges or by a silhouetted version of it?

If yellow color is used to resemble gold, then, grey or white can be used to resemble silver. Black and white or silhouetted version of Set Badge will give the feeling of something being "incomplete" which is the intention since the player didn't truly mastered the set, so, didn't deserve to win the real badge.

For Card Tooltip, it could use a grey dot followed by a Beaten datetime and all necessary data like normally happens to Mastered or Completed cases or different (I don't know how it is to supposed to behave to Beaten Games case though). However, I think something like: **"Congratulations, you have Beaten this Set! Now, try mastering it and win the real badge!"** as a message via HTML Title attribute would be nice since this wouldn't be appropriate to be shown on Card Tooltip when hovering on a Set Badge.

**Reply by Kinglink** [reply]
It does look like there's a beaten softcore.  I love the idea of "Silver" border for Beaten (Though a question will be Softcore mastery vs hardcore beaten, which should be shown.

However if the Silver Border is accepted for Beaten, there is a Beaten Softcore, In that case, I'd say have a faded/greyscale achievement icon.  

Maybe even let the beaten be a unique icon, that can be designed, but defaults to silver border/faded icons. 

The idea I have is essentially. 
Hardcore Mastery Gold as it is now. 
Hardcore Beaten Silver border, similar to Hardcore Mastery
Softcore Mastery Regular badge.
Softcore Beaten, greyscale/faded version of Regular badge. 

I know this is probably not the priority, but just sharing a possible idea.

**Reply by Player1041** [reply]
You could make the Softcore Mastery / Completion be bronze and make beaten be the current mastery badge, or a faded version of the badge

**Reply by suspect15** [reply]
There is often a significant difference in effort and skill between simply beating a game and mastering the achievement set.  First and foremost, RetroAchievements is an achievement centric community.  A badge is the most significant award that can be given and awarding it for simply beating a game diminishes the accomplishment of actually mastering the set in my opinion.  Simple borders are too indistinguishable, especially en masse, and this instead will afford imbalanced credit to profiles that aren't necessarily deserving considering the space in which RA is centered.

Creating even the slightest appearance of equivalency between beating and mastering amounts to a monumental shift in RA's primary mission.

**Reply by Kinglink** [reply]
If we are using that same logic where we want "mastery" to be special and recognized, we should remove Softcore Mastery Badges, because people can just cheat to their hearts content.  Is it really a major accomplishment if you use a game genie code to earn it?  Or more specifically master a Kaizo mario hack with invincibility and Jump in Mid Air turned on?  Those are currently allowed 

I'm not actually proposing removal of Softcore mastery badges, but I believe if we're worried about equivalency, at least personally I think a Hardcore "Beating" is closer to a Hard Core Mastery, than a "Softcore" Mastery is to a Hardcore mastery.

**Reply by suspect15** [reply]
The softcore discussion is moot because that ship sailed long ago, however RA essentially has two communities - the softcore and the hardcore.  There's not much cross over between the two of them so I don't think the parallel is quite there since beating games will apply to everyone.  Awarding badges for simply beating a game will really muddle up everyone's profiles.  Beating games is not something RA is intended to inherently value, think of it as statistic that is being tracked.  We are RetroAchievements, not RetroGames, RetroTracking, etc.  We have to keep a clear boundary between what the site is about and other nice/fun things to track or be aware of.

It's a very cool feature being added, but it shouldn't become a primary focus of the site.
